src/Grid.hpp

```diff
@@ -0,0 +1,104 @@
+//
+//  Grid.hpp
+//  Game of Life
+//
+//  Created by Alex Frasca on 01/05/2023.
+//
+#pragma once
+
+#include <SFML/Graphics.hpp>
+#include <vector>
+#include <cmath>
+#include <functional>
+
+namespace conway {
+
+//several values so we can draw nice colors as the cells die instead of just on/off
+enum class CELL_STATE {
+    CELL_OFF = 0,
+    CELL_LOW = 1,
+    CELL_MEDIUM = 2,
+    CELL_HIGH = 3,
+    CELL_ON = 4
+};
+
+//set colors for style
+inline sf::Color GoL_White          = sf::Color(235, 232, 249);
+inline sf::Color GoL_Red            = sf::Color(236, 63,  95 );
+inline sf::Color GoL_Fucshia        = sf::Color(215, 53,  180);
+inline sf::Color GoL_Blue           = sf::Color(110, 98,  173);
+inline sf::Color GoL_NavyBlue       = sf::Color(25,  35,  65 );
+inline sf::Color GoL_DarkFucshia    = sf::Color(148, 40,  84 );
+inline sf::Color GoL_Goldenrod      = sf::Color(247, 199, 74 );
+inline sf::Color GoL_Orange         = sf::Color(226, 93,  11 );
+inline sf::Color GoL_Black          = sf::Color(55,  49,  64 );
+inline sf::Color GoL_Periwinkle     = sf::Color(159, 138, 247);
+inline sf::Color GoL_Green          = sf::Color(81,  186, 155);
```

Comment:
Curious: why are these defined in `Grid.hpp`?

Reply:
It's fine for now, I'd consider moving these constants to a common header that other files can also use instead of just `Grid`.